### PR TITLE
A user can place a referral and is taken to the Find a Bedspace journey

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

Circled link now opens in a new tab

image

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[1. A user can place a referral and is taken to the Find a Bedspace journey](https://trello.com/c/R3yyMybc/1378-1-a-user-can-place-a-referral-and-is-taken-to-the-find-a-bedspace-journey)

File: server/views/temporary-accommodation/assessments/show.njk [hunk 1]
```diff
@@ -29,7 +29,11 @@
           <div class="moj-button-menu">
             <div class="moj-button-menu__wrapper">
               {% for action in actions %}
-                <a href="{{ action.href }}" role="menuitem" draggable="false" class="moj-button-menu__item {{ action.classes }}" data-module="govuk-button" data-secondary="true">
+                {% if action.newTab %}
+                  <a href="{{ action.href }}" role="menuitem" draggable="false" class="moj-button-menu__item {{ action.classes }}" data-module="govuk-button" data-secondary="true" rel="noreferrer noopener" target="_blank">
```
Comment (tahb) [inline server/views/temporary-accommodation/assessments/show.njk:33]: I know we didn't discuss this but I think we ought to consider changing the button text to be "Place referral (opens in a new tab)". I know it's long and will cause the actions list to become bigger but [it seems like the right thing to do](https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab)?

Comment (bryangaledxw) [reply server/views/temporary-accommodation/assessments/show.njk:33]: Ah yeah, I did see that. It's ugly, but yeah I think you're right!

**Comment by bryangaledxw** [comment]
@tahb Updated and ready for another look.

It doesn't look great. I don't look into styling the dropdown to be wider, but it started turning into a rabbit hole that I don't think we want to get into a the mo

image

**Review by tahb — APPROVED** [review]
LGTM

**Pull request merged**